src/vm/jitinterface.cpp

```diff
@@ -10090,24 +10090,28 @@ void CEEInfo::getEEInfo(CORINFO_EE_INFO *pEEInfoOut)
 
     JIT_TO_EE_TRANSITION();
 
-    InlinedCallFrame::GetEEInfo(&pEEInfoOut->inlinedCallFrameInfo);
+    if (!IsReadyToRunCompilation())
+    {
+        // inlinedCallFrameInfo is not used for R2R compilation
+        InlinedCallFrame::GetEEInfo(&pEEInfoOut->inlinedCallFrameInfo);
+    }
 
     // Offsets into the Thread structure
     pEEInfoOut->offsetOfThreadFrame = Thread::GetOffsetOfCurrentFrame();
```

Comment:
The Thread fields should be under the `!IsReadyToRunCompilation()` as well.

Reply:
Fixed. Also added zeroing memory for `inlinedCallFrameInfo` in R2R compilation